src/app/components/shared/menu/website-status-indicator/website-status-indicator.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, OnInit } from "@angular/core";
+import { WebsiteStatusService } from "@baw-api/website-status/website-status.service";
+import { websiteStatusMenuItem } from "@components/website-status/website-status.menu";
+import { withUnsubscribe } from "@helpers/unsubscribe/unsubscribe";
+import { StrongRoute } from "@interfaces/strongRoute";
+import { WebsiteStatus } from "@models/WebsiteStatus";
+import { takeUntil } from "rxjs";
+
+@Component({
+  selector: "baw-website-status-indicator",
+  template: `
+    <a
+      *ngIf="!isWebsiteHealthy"
+      class="block nav-link text-danger text-nowrap"
+      [strongRoute]="websiteStatusRoute"
+      [ngbTooltip]="
+        'The website is currently experiencing a temporary loss of services. ' +
+        'Some functionality may be reduced.'
+      "
+    >
+      <fa-icon [icon]="['fas', 'triangle-exclamation']"></fa-icon>
+    </a>
+  `,
+})
+export class WebsiteStatusIndicatorComponent
+  extends withUnsubscribe()
+  implements OnInit
+{
+  public constructor(private api: WebsiteStatusService) {
+    super();
+  }
+
+  protected isWebsiteHealthy = true;
+  protected websiteStatusRoute: StrongRoute = websiteStatusMenuItem.route;
+
+  public ngOnInit(): void {
+    this.api
+      .show()
+      .pipe(takeUntil(this.unsubscribe))
+      .subscribe((status: WebsiteStatus) => {
+        this.isWebsiteHealthy = status.isStatusHealthy;
+      });
```

Comment:
Does this component get reloaded on router navigation? I don't think it does right, since it is outside the router outlet.

Otherwise if the api enters a degraded state while they're on the page then this won't every update.

Probably need an observeable in the service that requests a new status object every 30 seconds or so.